src/commands/env/env.ts

```diff
@@ -38,10 +39,11 @@ export const createEnvCommand = (program: BaseCommand) => {
     .command('env:import')
     .argument('<fileName>', '.env file to import')
     .option(
-      '-r, --replace-existing',
+      '-R, --replace-existing',
```

Comment:
undo this one

Reply:
What does this mean? Is this a note to self or you noting that this reverts changes from a previous PR?